batcher/aligned/src/main.rs

```diff
@@ -107,20 +109,87 @@ pub struct SubmitArgs {
     keystore_path: Option<PathBuf>,
     #[arg(name = "Private key", long = "private_key")]
     private_key: Option<String>,
-    #[arg(
-        name = "Max Fee (ether)",
-        long = "max_fee",
-        default_value = "0.0013ether" // 13_000 gas per proof * 100 gwei gas price (upper bound)
-    )]
-    max_fee: String, // String because U256 expects hex
+    #[command(flatten)]
+    price_estimate: PriceEstimateArgs,
     #[arg(name = "Nonce", long = "nonce")]
     nonce: Option<String>, // String because U256 expects hex
     #[arg(
         name = "The working network's name",
         long = "network",
         default_value = "devnet"
     )]
-    network: NetworkArg,
+    network: Network,
+}
+
+#[derive(Args, Debug)]
+#[group(required = false, multiple = false)]
+pub struct PriceEstimateArgs {
```

Comment:
You should try to stick to chosen names, you have mostly used `fee` but now you are using `price`.